lib/src/method_channel.dart

```diff
@@ -8,7 +8,8 @@ class FlutterDeviceLocaleMethodChannelPlugin
     extends FlutterDeviceLocalePlatform {
   @override
   Future<List<String>> deviceLocales() async {
-    final List<dynamic> result = await _channel.invokeMethod('deviceLocales');
+    final result =
+        await (_channel.invokeMethod('deviceLocales') as Future<List<dynamic>>);
```

Comment:
This should be

`final result = (await _channel.invokeMethod('deviceLocales') as List<dynamic>);`

Fixes https://github.com/bratan/flutter_translate/pull/63#issuecomment-815762314

Reply:
Thanks very much